eclipse-collections-code-generator/src/main/resources/test/bag/immutable/immutablePrimitiveEmptyBagTest.stg

```diff
@@ -41,10 +43,10 @@ public class Immutable<name>EmptyBagTest extends AbstractImmutable<name>BagTestC
     }
 
     @Override
-    @Test(expected = ArithmeticException.class)
+    @Test
     public void average()
     {
-        this.classUnderTest().average();
+        assertThrows(ArithmeticException.class, () -> this.classUnderTest().average() );
```

Comment:
There are a handful of places like this with a space before a closing paren that are likely going to be CheckStyle warnings. Otherwise, the whole thing LGTM! Thanks!

Reply:
For some reason the style check is not failing on my computer, making this annoying to fix.  Working on it.

Reply:
OK fixed now.

Reply:
Well I *thought* it was fixed, but apparently not.